api/app/signals/apps/users/admin.py

```diff
@@ -36,11 +37,29 @@ def departments(self, obj):
     departments.short_description = 'Afdeling(en)'
     departments.allow_tags = True
 
+    @admin.action(description='Download CSV')
```

Comment:
👍🏼 nice, this replaces the `func.short_description = 'short description'`. There are other places where the `short_description` is used, perhaps a refactor PR can be made?

Reply:
Yeah sounds good, I saw that this was new as per Django 3.2, it also makes it a bit more clear that this method is for the action that is specified in the `actions` property.